datafusion/physical-expr/src/window/built_in.rs

```diff
@@ -97,6 +97,10 @@ impl BuiltInWindowExpr {
 }
 
 impl WindowExpr for BuiltInWindowExpr {
+    fn func_name(&self) -> Result<&str> {
+        not_impl_err!("function name not determined")
```

Comment:
why wouldn't we implement func_name for a built in window function 🤔

Reply:
The reason is because I don't need it -- for name checking in `nullable`